frontend/src/metabase/query_builder/components/view/QuestionRowCount.jsx

```diff
@@ -31,7 +31,11 @@ const QuestionRowCount = ({
   const limitMessage =
     query instanceof StructuredQuery
       ? query.limit() == null || query.limit() >= HARD_ROW_LIMIT
-        ? cappedMessage
+        ? typeof result.row_count === "number"
```

Comment:
How hard was it for you to understand this ternary chain?

Would you think it benefitial to take the chance and break it down?

Reply:
It definitely took me longer to fix than necessary due to the mental overhead. I'm categorically biased against nested ternary operators but I decided to follow the existing style.

Pushed a commit replacing both nested ternary operators with if blocks. Let me know if I should revert one or either of them back to the nested ternary operators